### PR TITLE
Embedding Images and Video

### PR DESCRIPTION
**Pull request by leolpaz**
We made a clone of the youtube player page, it has a functional video, buttons, and interactive icons. The page works best on a 1366x768 resolution and using the firefox browser, assets might look out of place using other browsers or resolutions. It has mock suggestions showing video name and views, and a mock section for commentaries. 

The focus of this branch was to make everything fit in their right places, along with a functional youtube video player.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+}
+
+
+main {
+  padding-left: 8%;
+  padding-right: 7%;
+  padding-top: 1.5%;
+  margin-top: 0%;
+  background-color: #f9f9f9;
+  margin-left: 0%;
```
Comment (jubaan) [inline style.css]: See issue above about the 0 and the units

**Review by jubaan — CHANGES_REQUESTED** [review]
# This review follows the required changes in PR #2 

# Status: Changes Required :recycle: 

## :wave: :wave: Hello @leolpaz and @gasb150  . Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] The project has a conflict that must be resolved
image

- [x] The extra space haven't been removed

> * [x]  Please move this down and remove extra space
> 
> **clone**
> 
> chrome-capture (5)

- [x] Values of 0 shouldn't have units specified. (this issue still appear on the CSS). See inline

### [Comments] :green_circle: 

- :rotating_light: :rotating_light: :rotating_light: The PR should be the same that has been reviewed before, this is not the first time it has happened, please all corrections should be in the initial PR #1. Don't make this a habbit. :rotating_light: :rotating_light: :rotating_light: 

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: style.css [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+
+
+.video-information {
+  display: flex;
+  flex-direction: column;
+}
+
+
+.video {
+  margin: 0px;
```
Comment (ElijahTobs) [inline style.css]: [Mandatory] Kindly make sure that you don't add units to values of zero

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required! :recycle:  
### Good job @leolpaz and @gasb150 . You're almost there :smile: 

## Required Changes :wrench: 
### [Mandatory]

- [x] Zero CSS values should not carry units.

### [Optional]


- [x] I think it will be better if you replaced **Author1** and **Author2** (as you have it in your README.md file) with your respective names.
author1

#### Implement the changes and submit for another review :wink: 
### Happy Coding :muscle: :man_technologist:

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://raw.githack.com/leolpaz/YoutubePlayerPageClone/YoutubePage/Index.Html) shows a good understanding of Embedding Images and Video. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @leolpaz and @gasb150! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know on slack at any time. I will be happy to help.**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**